coralproject/talk
# External Signin for Admin

## What does this PR do?

- Adds a slot to support different administrative login strategies.
- Fixes bug with `postMessage` service.

## How do I test this PR?

1. Enable google-auth
2. Visit admin and notice the "Sign in with Google" button
